bigbluebutton-html5/imports/startup/server/index.js

```diff
@@ -1,17 +1,48 @@
 import { Meteor } from 'meteor/meteor';
+import { WebAppInternals } from 'meteor/webapp';
 import Langmap from 'langmap';
 import Users from '/imports/api/users';
 import fs from 'fs';
 import './settings';
 import Logger from './logger';
 import Redis from './redis';
 
+var parse = Npm.require('url').parse;
+
 const AVAILABLE_LOCALES = fs.readdirSync('assets/app/locales');
 
 Meteor.startup(() => {
   const APP_CONFIG = Meteor.settings.public.app;
   const env = Meteor.isDevelopment ? 'development' : 'production';
-  Logger.warn(`SERVER STARTED. ENV=${env}, nodejs version=${process.version}`, APP_CONFIG);
+  const CDN_URL = APP_CONFIG.cdn;
+
+  if (CDN_URL) {
+    // Add CDN
+    BrowserPolicy.framing.disallow();
```

Comment:
We're not going to be able to use this. There are some people that will embed the client in their apps.